zh_cn/02.安装与配置/03.helm.md

```diff
@@ -47,12 +49,15 @@ helm upgrade --install zadig --namespace zadig  koderover-chart/zadig --set endp
 > IP 为 Kubernetes 任一节点的外网可访问 IP。
 
 ```bash
-helm upgrade --install zadig --namespace zadig  koderover-chart/zadig --set endpoint.type=IP \
+helm upgrade --install zadig --namespace <NAMESPACE> koderover-chart/zadig --set endpoint.type=IP \
     --set endpoint.IP=<IP> \
     --set ingress-nginx.controller.service.nodePorts.http=<30000~32767任一端口>
-# v1.7.0+ 版本需加上以下参数
-#   --set dex.config.issuer=http://<IP>:<PORT>/dex
+# v1.7.0+ 版本需加上以下参数，且确保 <NAMESPACE> 和安装 Zadig 系统的 Namespace 一致
+#   --set global.extensions.extAuth.extauthzServerRef.namespace=<NAMESPACE>
+#   --set dex.fullnameOverride=zadig-<NAMESPACE>-dex
+#   --set dex.config.issuer=http://zadig-<NAMESPACE>-dex/dex
```

Comment:
--set dex.config.issuer=http://zadig-namespace-dex:5556/dex

Reply:
GOT. `zh_cn` will be deleted for supporting multi versions. I will fix this comment after  https://github.com/koderover/zadig-doc/pull/162 merged. 

Otherwise, there will be conflicts in https://github.com/koderover/zadig-doc/pull/162